docs/installation/troubleshoot.md

```diff
@@ -64,6 +64,8 @@ Run `bundle install` again
 
 If neither of these works, try exploring [this StackOverflow link](http://stackoverflow.com/questions/23536893/therubyracer-gemextbuilderror-error-failed-to-build-gem-native-extension)
 
+Another error that may occur on macOS when running `bundle install` is that the dependancy mimemagic may fail to install, causing `bundle install` to fail. A possible fix is if you have homebrew installed, run `brew install shared-mime-info` and then run `bundle install`. For more information, read the follow [github issues link](https://github.com/mimemagicrb/mimemagic/issues/162).
```

Comment:
Nit, feel free to reorganize the previous sections by the Gem that is resulting in failures (i.e use actual headers for eventmachine, libv8, and mimemagic). The way our current doc uses code blocks to demarcate the various failures in this section is not idiomatic and also causes it not to show up on the left navbar of the documentation. Using headers helps with ease of searching (appears on navbar) and helps with organization